src/localauthentication.cs

```diff
@@ -88,12 +93,194 @@ interface LAContext {
+		LAAuthenticationRequirement DefaultRequirement { get; }
+
+		[Static]
+		[Export ("biometryRequirement")]
+		LAAuthenticationRequirement BiometryRequirement { get; }
+
+		[Static]
+		[Export ("biometryCurrentSetRequirement")]
+		LAAuthenticationRequirement BiometryCurrentSetRequirement { get; }
+
+		[Static]
+		[Export ("biometryRequirementWithFallback:")]
+		LAAuthenticationRequirement GetBiometryRequirement (LABiometryFallbackRequirement fallback);
+	}
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+	[BaseType (typeof (NSObject))]
+	interface LABiometryFallbackRequirement
+	{
+		[Static]
+		[Export ("defaultRequirement")]
+		LABiometryFallbackRequirement DefaultRequirement { get; }
+
+		[Static]
+		[Export ("devicePasscodeRequirement")]
+		LABiometryFallbackRequirement DevicePasscodeRequirement { get; }
+	}
+
+	delegate void LARightAuthorizeCompletionHandler ([NullAllowed] NSError error);
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+	[BaseType (typeof (NSObject))]
+	interface LARight
+	{
+		[Export ("state")]
+		LARightState State { get; }
+
+		[Export ("tag")]
+		nint Tag { get; set; }
+
+		[Export ("initWithRequirement:")]
+		NativeHandle Constructor (LAAuthenticationRequirement requirement);
+
+		[Async]
+		[Export ("authorizeWithLocalizedReason:completion:")]
+		void Authorize (string localizedReason, LARightAuthorizeCompletionHandler handler);
+
+		[Async]
+		[Export ("checkCanAuthorizeWithCompletion:")]
+		void CheckCanAuthorize (LARightAuthorizeCompletionHandler handler);
+
+		[Async]
+		[Export ("deauthorizeWithCompletion:")]
+		void Deauthorize (Action handler);
+	}
+
+	delegate void LARightStoreCompletionHandler ([NullAllowed] LAPersistedRight right, [NullAllowed] NSError error);
+	delegate void LARightStoreRemoveRightCompletionHandler ([NullAllowed] NSError error);
+
+	[Mac (13,0), iOS (16,0), MacCatalyst (16,0), NoWatch, NoTV]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface LARightStore
+	{
+		[Static]
+		[Export ("sharedStore")]
+		LARightStore SharedStore { get; }
+
+		[Async]
+		[Export ("rightForIdentifier:completion:")]
+		void GetRight (string identifier, LARightStoreCompletionHandler handler);
+
+		[Async]
+		[Export ("saveRight:identifier:completion:")]
+		void SaveRight (LARight right, string identifier, LARightStoreCompletionHandler handler);
+
+		[Async]
+		[Export ("saveRight:identifier:secret:completion:")]
+		void SaveRight (LARight right, string identifier, NSData secret, LARightStoreCompletionHandler handler);
+
+		[Async]
+		[Export ("removeRight:completion:")]
+		void RemoveRight (LAPersistedRight right, LARightStoreRemoveRightCompletionHandler handler);
+
+		[Async]
+		[Export ("removeRightForIdentifier:completion:")]
+		void RemoveRight (string identifier, LARightStoreRemoveRightCompletionHandler handler);
+
+		[Async]
+		[Export ("removeAllRightsWithCompletion:")]
+		void RemoveAllRights (LARightStoreRemoveRightCompletionHandler handler);
```

Comment:
It is a Right store and all methods have the Right word. Should we remove that since it seems redundant.